app/uk/gov/hmrc/pegaproofofconceptfrontend/controllers/InputController.scala

```diff
@@ -48,8 +52,13 @@ class InputController @Inject() (
         Future.successful((BadRequest(views.stringInputPage(formWithErrors))))
       },
       (validFormData: StringInputForm) => {
-        logger.info(s"SUBMITTED STRING: '${validFormData.string}' TO PEGA")
-        Future.successful(Redirect(uk.gov.hmrc.pegaproofofconceptfrontend.controllers.routes.InputController.getStringInput))
+        pegaProxyConnector.submitPayloadToProxy(Payload.fromStringInputForm(validFormData)).map {
+          case response if response.status === 200 => {
+            logger.info(s"SUBMITTED STRING: '${validFormData.string}' TO PEGA")
+            Ok(views.fakePegaPage())
+          }
+          case _ => InternalServerError(errorHandler.internalServerErrorTemplate)
```

Comment:
could we get an error message here to say what the response status and body was? It'd help with debugging if we ever get here